homeassistant/components/media_player/__init__.py

```diff
@@ -523,7 +521,36 @@ def source_list(self):
 
     @property
     def supported_media_commands(self):
-        """Flag media commands that are supported."""
+        """Flag media commands that are supported.
+
+        DEPRECATED: Included for temporary custom platform compatibility.
+        """
+        return None
+
+    @property
+    def supported_features(self):
+        """Flag media player features that are supported."""
+        # Begin temporary transition logic
+
+        if self.supported_media_commands is not None:
+            # If this platform is still using supported_media_commands, issue
+            # a logger warning once with instructions on how to fix it.
+            if not getattr(self, '_supported_features_warned', False):
+                import inspect
```

Comment:
THe first access would be really heavy as it would import a new module. Can't this warning move somewhere else?